0.0.6

```diff
@@ -74,14 +74,14 @@
             "description": "Add Type field to AC Task Layout",
             "taskList": [
                 {
-                    "jiraTicket": "ac-57a",
+                    "jiraTicket": "ac-57a,ac-1878",
```

Comment:
We shouldn't be putting two tickets in that do the same thing. Resolve the overlap in JIRA with a PM by consolidating or deleteing duplicat tickets. Otherwise QA will have to lookup two tickets during debug, causing more confusion.

Reply:
What he said^